### PR TITLE
Add robotframework-lsp dependency

### PR DESCRIPTION
**Pull request by jgarciao**
In order to use PyCharm's [Robot Framework Language Server Plugin](https://plugins.jetbrains.com/plugin/16086-robot-framework-language-server) we need this lib

### CODE REVIEW

**Comment by lugi0** [comment]
Aren't the first two commits already merged into master? You might want to fetch and rebase

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+
+- Configure name and email in git:
+
+  ```
+  git config --global user.name "Your Name"
```
Comment (vasukulkarni) [inline CONTRIBUTING.md:9]: we already merged this one?

File: setup.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
         "reportportal-client",
         "robotframework>=4",
         "robotframework-debuglibrary",
+        "robotframework-lsp",
```
Comment (vasukulkarni) [inline setup.py:20]: this can be moved to requirements-dev.txt as its required for development work but not for regular sanity/regressions.

**Comment by jgarciao** [comment]
Closing this PR, as it's replaced by #48

**Pull request closed without merging**